src/app/view/perspectives/post-it/document/post-it-document.component.html

```diff
@@ -1,135 +1,65 @@
-<div class="card-header d-inline-flex py-1 px-3">
-  <span class="mr-1 d-flex overflow-hidden"
-        [style.color]="collection?.color"
-        [title]="collection?.name">
-    <i class="icon fa-fw {{ collection?.icon }} mr-1"></i>&nbsp;
-    <span class="collection-name">
-      {{ collection?.name }}
-    </span>
-  </span>
-
-  <div class="text-right flex-fill align-self-center">
-    <ng-container *ngIf="hasWriteRole()">
-      <i class="button text-danger fas fa-trash rounded p-1"
-         title="Remove Record" i18n-title="@@document.button.remove"
-         (click)="onRemove()"></i>
-      <i class="button text-success fas fa-pencil rounded p-1"
-         title="Edit Record" i18n-title="@@document.button.edit"
-         (click)="selectionHelper.setEditMode(true); selectionHelper.select(0, 0, postItModel)"></i>
-    </ng-container>
-    <i *ngIf="postItModel.initialized"
-       class="text-warning fa-star button rounded p-1"
-       [class.always-visible]="postItModel.document.favorite"
-       title="{{ postItModel.document.favorite ?  'Remove from favorites' : 'Add to favorites'}}"
-       i18n-title="@@collection.document.favorite"
-       [ngClass]="postItModel.document.favorite ? 'fas' : 'far'"
-       (click)="toggleFavorite()"></i>
-  </div>
-</div>
+<post-it-document-header
+  class="card-header px-2 py-1 d-flex"
+  [collection]="collection"
+  [readonly]="!hasWriteRole"
+  [favorite]="getFavorite$() | async"
+  [initialized]="documentModel.id"
+  (remove)="onRemove()"
+  (toggleFavorite)="onToggleFavorite()"
+  (edit)="onEdit()">
+</post-it-document-header>
 
 <div class="card-body p-2">
   <ul class="m-0 p-0" #content>
+    <ng-container *ngFor="let row of getRows$() | async; trackBy:getTrackBy(); let idx = index">
```

Comment:
I am afraid this will generate a new subscription on every change detection cycle. Because `getRows$()` is a function and not class property.

Reply:
What about using `share` operator?